automatic/composer/update.ps1

```diff
@@ -0,0 +1,37 @@
+﻿import-module au
+
+$releases = 'https://github.com/composer/windows-setup/releases'
+
+function global:au_SearchReplace {
+  @{
+    ".\tools\chocolateyInstall.ps1" = @{
+      "(^[$]fileName\s*=\s*)('.*')"     = "`$1'$($Latest.FileName32)'"
+      "(?i)(^\s*checksum\s*=\s*)('.*')" = "`$1'$($Latest.Checksum32)'"
+    }
+
+    "$($Latest.PackageName).nuspec" = @{
+      "(\<releaseNotes\>).*?(\</releaseNotes\>)" = "`${1}$($Latest.ReleaseNotes)`$2"
+    }
+
+    ".\legal\VERIFICATION.txt"      = @{
+      "(?i)(\s+x32:).*"     = "`${1} $($Latest.URL32)"
+      "(?i)(checksum32:).*" = "`${1} $($Latest.Checksum32)"
```

Comment:
Should also be able to replace the checksum type here (when added to the `VERIFICATION.txt` file).

Reply:
Done